<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchBook</class>
 <widget class="QMainWindow" name="searchBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>733</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>mainWnd</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>app.gif</normaloff>app.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tvBase">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>771</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFilter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите заглавие или автора книги или инвентарный номер (регистр значения не имеет)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFilter">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>-6</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Запрос</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTableFilter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Окно поиска</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbProgress">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>660</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgress">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>680</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обработка запроса</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFilial">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>29</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>56</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>По Заголовку</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbAutor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>56</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>По Автору</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbID">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>56</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>По Инв. номеру</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHelpNeed">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>6</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Нужна помощь?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbFullInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>771</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Полная информация</string>
    </property>
    <widget class="QLabel" name="lblFil">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>96</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Место хранения:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leKnowledgeSection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>76</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblYear">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Год:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAutor">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblKnowledgeSection">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Раздел:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblID">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Инвентарный номер:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leYear">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblAutor">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Автор:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Заглавие:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSeries">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>132</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Серия:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSeries">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>148</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="leBookPart">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>148</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblBookPart">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>132</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кн./Том/ч.:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leISBN">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>148</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblISBN">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>132</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ISBN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSysCode">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>132</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Сист. шифр:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSysCode">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>148</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="leID">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>76</y>
       <width>591</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans Mono'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="leFil">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>112</y>
       <width>751</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbDoubleSearch">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>53</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblHelp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Для поиска книг выберите поле, по которому необходимо найти книгу (заголовок, автор, инвентарный номер).&lt;br/&gt;Для уточнения запроса - можно выбрать филиал из выпадающего списка.&lt;br/&gt;Для одновременного поиска по заголовку и автору - нажмите на кнопку + (плюс) и вводите поиск в следующем формате: &lt;span style=&quot; font-weight:600;&quot;&gt;Фамилия_Автора % часть_заглавия_книги&lt;/span&gt;. Например: &lt;span style=&quot; font-weight:600;&quot;&gt;пуш % есе&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFullInfo">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Полная информация</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfoAboutUpdateBases">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>5</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация о актуальности базы</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
